release_notes.md

```diff
@@ -5,10 +5,7 @@
 - Update Python Worker Version to [4.5.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.5.0)
 
 - Host support for out-of-proc cancellation tokens ([#2153](https://github.com/Azure/azure-functions-host/issues/2152))
-- Update Python Worker Version to [4.4.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.4.0)
```

Comment:
are these new updates ? or should this be empty?

Reply:
Good question - they are new! see https://github.com/Azure/azure-functions-host/compare/v4.10.1...dev